spec.bs

```diff
@@ -2792,6 +2834,23 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+The {{Fence/notifyEvent}} method of the {{Fence}} interface introduces a new event type, <code><dfn>fencedtreeclick</dfn></code>.
+
+The <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects">
+event handlers on elements, Document objects, and Window objects</a> section of [[HTML]] is modified to 
+include a new row. The [=event handler=] name is <code>onfencedtreeclick</code>, and the [=event handler event type=] is
```

Comment:
It'd be best to replicate the style of table that you see in https://html.spec.whatwg.org/multipage/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects. Just have a single row pointing to the stuff. you want.

### Event handler definition
Note that the "event handler" column should include a definition for `onfencedtreeclick`, and the "event handler event type" should be a _link_ (not dfn) to the new `fencedtreeclick` event type.

### Event type definition
Then the `fencedtreeclick` event type should be defined in a new section of this spec that monkeypatches the table you find in https://html.spec.whatwg.org/C#events-2, which is responsible for defining all of the event types (or linking to them when another spec defines them; but since we're monkeypatching HTML here, we want to add the dfn to that table).

### Misc

You'll also need to monkeypatch this list (https://html.spec.whatwg.org/multipage/dom.html#global-attributes%3Aevent-handler-content-attributes) to include a link to the `onfencedtreeclick` event handler, which is defined in the table I mention in "Event handler definition" section above.

Reply:
I believe I've got all of this addressed, PTAL!